.github/CONTRIBUTING.md

```diff
@@ -89,6 +89,8 @@ implementing features, refactoring code, porting to a different language),
 otherwise you risk spending a lot of time working on something that the
 project's developers might not want to merge into the project.
 
+If possible, have **PowerPoint 2010, 2013, 2016** installed on your computer and run tests on all of them before submitting the PR. Your changes are expected to work on all of these versions.
```

Comment:
Matches issue description to include requirement to install **PowerPoint 2010, 2013, 2016**. Not sure if it will be better to be explicit that having 1 version is minimally sufficient.